plugin.rb

```diff
@@ -51,6 +52,12 @@
       schedule_jobs(post)
   end
 
+  on(:user_created) do |user|
+    if user.oauth2_user_infos.blank?
+      user.update!(username: Communitarian::UniqueUsername.new(user).to_s)
```

Comment:
May be do raise error on Logger if something went wrong?

Reply:
@Vadim-Kurnatovskiy  we automatically will see error in https://test-arts.online/logs if username will not be updated